spec/ruby_llm/active_record/acts_as_spec.rb

```diff
@@ -70,25 +70,64 @@ def execute(expression:)
     end
   end
 
-  it 'persists chat history' do # rubocop:disable RSpec/ExampleLength,RSpec/MultipleExpectations
-    chat = Chat.create!(model_id: 'gpt-4o-mini')
-    chat.ask("What's your favorite Ruby feature?")
-
-    expect(chat.messages.count).to eq(2)
-    expect(chat.messages.first.role).to eq('user')
-    expect(chat.messages.last.role).to eq('assistant')
-    expect(chat.messages.last.content).to be_present
-    expect(chat.messages.last.input_tokens).to be_positive
-    expect(chat.messages.last.output_tokens).to be_positive
```

Comment:
why removing this very important test?

Reply:
Yes, adding that back was a mistake. I was moving quickly yesterday.